wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -344,7 +344,7 @@ private void cropPhotoTest() throws Exception {
         String testTag = "crop";
 
         // To improve travel accuracy perform the slide bar operation slowly
-        final int steps = 500;
+        final int steps = 100;
```

Comment:
The comment suggests this was to improve accuracy, have you tested this on multiple devices to make sure its not an issue?